packages/compass/package.json

```diff
@@ -160,7 +160,7 @@
     "email": "compass@mongodb.com"
   },
   "dependencies": {
-    "@mongosh/node-runtime-worker-thread": "^1.5.0",
+    "@mongosh/node-runtime-worker-thread": "^1.5.1",
```

Comment:
Can you bump driver and the other shell versions as well, similar to a8ff75c22ef7570375e565866f7378888457e11a? Otherwise shell versions are out of sync and we’d be running different driver versions in Compass and mongosh.